cherrypy/test/test_static.py

```diff
@@ -215,13 +215,13 @@ def test_config_errors(self):
         self.assertErrorPage(500)
         if sys.version_info >= (3, 3):
             errmsg = (
-                'TypeError: staticdir\(\) missing 2 '
+                r'TypeError: staticdir\(\) missing 2 '
```

Comment:
Current line isn't a docstring, we were talking about other places touched by this PR. Nevermind :)